### DISCUSSION TITLE
🖼 Fanart Competition #5

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#3011] [Art]
For this competition, we encourage a wide range of art styles and techniques, not limited to pixel art. Feel free to explore your creativity and express yourself in any art style you choose!

Before submitting your artwork, we highly recommend that you read our fanart guidelines, which can be found here:

https://docs.sunflower-land.com/contributing/competitions/fan-art-competition

## Theme
The theme for this month is: **Halloween** 🎃

Show us your creativity by designing something related to the chosen theme! Whether it's a character that you love, a scene that captivates you, or a moment that leaves a lasting impression, we want to see what you can come up with. Let your imagination soar and share your unique perspective with us! ❤

## Requirements ✏️

- AI-generated art will not be accepted.
- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting art, please include the following:

- Your Farm ID.
- Your Discord ID.
- A image of your art with good quality.
- A NPC that represents you. 
- A Google Drive link with proof of your creation. 
To verify the authenticity of your artwork, we kindly request that you provide proof of your creative process. This may include work-in-progress images, concept sketches, time-lapse videos, and, in the case of digital art, we also request the original files.

Please note that the inclusion of NPCs will not have any impact on the judging process of the competition. However, if you choose to provide an NPC, it will be displayed in-game alongside your art piece. If you do not include an NPC, a random one will be selected. Also, a Bumpkin version of your NPC would be appreciated, but not required. 

You can use this image to showcase your NPC:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/81132ce6-4863-4be7-a3ca-b5f78434ee60)

## Rewards 🏆
The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however not only will the winning artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

We also want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅
Artists have until **October 29th** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

So, what are you waiting for? Grab your brushes, pens, or digital tools and get creating! Show us what you're made of and join our community of talented artists in celebrating the vibrant world of Sunflower Land Game. We can't wait to see what you come up with!

Good luck to all participants! 🎉

**Comment by davosalm** [comment]
My first submission! Here comes the Cthulhu, friend of occultist goblins. He's a good friend of the blacksmith, as sometimes his lamp gets a little rusty. 

Ps: I've used a color that is similar to the background on the lamp, so it looks "empty". lol

![CTHULHU_Caramelaikos-export](https://github.com/sunflower-land/sunflower-land/assets/9125640/081f671a-602f-4d0f-8903-d024ae71df70)
![CTHULHU_Caramelaikos](https://github.com/sunflower-land/sunflower-land/assets/9125640/cb335465-4c00-4349-9c27-43647e35adf7)

![pixil-gif-drawing](https://github.com/sunflower-land/sunflower-land/assets/9125640/20ddbf25-aca4-4376-a340-bc6452b7a44d)

Proof of work:

https://drive.google.com/drive/folders/1Ss2-1-5hC2hHiMD2hqJIOwLu7JtwCTOX?usp=sharing

The lantern I've made on pixilart platform (online, through smartphone) and the Bumpkin was made with Aseprite. Both archives are inside the link.

The little .gif was made with pixilart, but somehow I closed the tab without saving the project and only got the archive (and it can be seen on my personal profile, too: https://www.pixilart.com/photo/bumpkingif-6dfd9876af3bcf4) D:


Farm ID: 99994
Discord: Caramelaikos

**Comment by esp123** [comment]
Scary Mike
Desc: T Posting Mike the Scarecrow scaring some crows 🤣 . Created from Pixel Studio (PC and Mobile).

Art Progression:
![TpostingMike](https://github.com/sunflower-land/sunflower-land/assets/74337946/e570c76b-90a5-4af1-893a-7008e98f4e43)

Final:
![poorcrows](https://github.com/sunflower-land/sunflower-land/assets/74337946/2f95123e-2fc9-4bc8-80a1-18278e8ff02b)

File:
Github doesn't support .psp file. 😞 

Farm ID: 80399
Discord: deeefault

Npc:
![npc](https://github.com/sunflower-land/sunflower-land/assets/74337946/691a0452-5bd5-4f0b-a7f1-a43479c79f8d)

Bumpkin:
![marioooo_2](https://github.com/sunflower-land/sunflower-land/assets/74337946/d2c2bbd0-6a28-45df-be5a-6bd89746bad8)

**Comment by IceMonky29** [comment]
**Scary ghost and strange spider pumpkin, I hope you like it**

![Final](https://github.com/sunflower-land/sunflower-land/assets/142521805/1af7cd46-c9ec-452c-b28d-281b96550adb)

**tests in google drive**
https://drive.google.com/drive/folders/1J7zb-0oZhsyZHx37QCo_Xumun_h7GCXj?usp=drive_link



**Farm: 162510**
 **ID Discord: icemonky**

I'm improving little by little hahahahah

**Comment by andand0** [comment]
### Trick or Fruit

<hr>


_Be careful when you harvest your fruits on Halloween._

![sunflower_haloweeen_6](https://github.com/sunflower-land/sunflower-land/assets/142132586/e1cc18ce-c226-4890-93bf-93ffde701bac)


<br>

Google Drive link: [Sunflower fanart #5 trick or fruit](https://drive.google.com/drive/folders/1i_azLMZ1BOAwJU0wQjdkaRPSNPuqWEyp?usp=drive_link)

<br>

**My NPC**

| Gif| Sprite |
|--------|--------|
| ![Andando-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/8f46d08d-bd88-4c37-a35b-0fab4de97d46)| ![Andando-NPC-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/055f814a-48e9-4a22-9fb3-ab47f7f9f474) | 

`Farm ID: 165810`
`Discord: Andando`

**Comment by Vergelsxtn** [comment]
### **FIND THE IMPOSTOR😈**

If you can't beat them, Join them (only maze player will know)
gif
![halloween2-fanart-final3](https://github.com/sunflower-land/sunflower-land/assets/99605039/ba4820ec-aaac-4d17-9a9e-d711331a74ba)
png
![halloween2-fanart-final3](https://github.com/sunflower-land/sunflower-land/assets/99605039/36a3b2c1-9f70-417e-bf95-e545210bd6a3)

A NPC that represents me:
![plague-doctor](https://github.com/sunflower-land/sunflower-land/assets/99605039/01826f58-09c6-4027-bff3-6010a3fbaf94)

A Google Drive link with proof of my creation: [Click here](https://drive.google.com/drive/folders/1KZ7qOLgo0TtnEF6ifgmKMSbliL_QLWEW?usp=drive_link)

My Farm ID: 63
My Discord ID: Vergelsxtn

**Happy Halloween!!!🎃🦇👻**
